experimental/logical-assignment-operators.md

```diff
@@ -0,0 +1,30 @@
+# [Logical Assignment Operators][proposal-logical-assignment]
+
+## Expressions
+
+### LogicalAssignmentExpression
+
+```js
+interface LogicalAssignmentExpression <: Expression {
+    type: "LogicalAssignmentExpression";
+    operator: LogicalAssignmentOperator;
+    left: Pattern | Expression;
```

Comment:
I don't think `Pattern` is allowed here.

Reply:
Correct.

Reply:
Nice catch.